src/index.ts

```diff
@@ -45,6 +47,7 @@ const ALWAYS_SHOW_ERRORS = new Set(['react', 'react-dom']);
 const cwd = process.cwd();
 const banner = chalk.bold(`snowpack`) + ` installing... `;
 const installResults = [];
+let dependencyInfo: DependencyInfoOutput = null;
```

Comment:
nit: is `DependencyInfoOutput` output, or output info? May be better named as `DependencyOutputInfo` or `DependencyOutputStats`.